packages/browser/src/Session.ts

```diff
@@ -120,16 +115,19 @@ export async function silentlyAuthenticate(
     // on incoming redirect. This way, the user is eventually redirected back
     // to the page they were on and not to the app's redirect page.
     window.localStorage.setItem(KEY_CURRENT_URL, window.location.href);
-    await clientAuthn.login({
-      sessionId,
-      prompt: "none",
-      oidcIssuer: storedSessionInfo.issuer,
-      redirectUrl: storedSessionInfo.redirectUrl,
-      clientId: storedSessionInfo.clientAppId,
-      clientSecret: storedSessionInfo.clientAppSecret,
-      tokenType: storedSessionInfo.tokenType ?? "DPoP",
-      inIframe: options.inIframe,
-    });
+    await clientAuthn.login(
+      {
+        sessionId,
+        prompt: "none",
+        oidcIssuer: storedSessionInfo.issuer,
+        redirectUrl: storedSessionInfo.redirectUrl,
+        clientId: storedSessionInfo.clientAppId,
+        clientSecret: storedSessionInfo.clientAppSecret,
+        tokenType: storedSessionInfo.tokenType ?? "DPoP",
+        inIframe: options.inIframe,
+      },
+      this
```

Comment:
My bad, this should not be `this`, but instead a `Session` instance should be passed to the `silentlyAuthenticate` function, meaning that calls to `silentlyAuthenticate` will need to be passed `this` if called from a Session object.